tests/tests.py

```diff
@@ -19,20 +18,23 @@ def setUp(self):
         articles_cnt = 4
         users_cnt = 10
 
-        def mock_process(self):
+        def mock_process(self=None):
```

Comment:
In the future, we should remove 'self' totally. but I'm still depending on Python 2 sometimes.